src/Commands/ProjectCreateCommand.php

```diff
@@ -27,43 +27,41 @@
 use Pantheon\TerminusBuildTools\ServiceProviders\ProviderEnvironment;
 use Pantheon\TerminusBuildTools\ServiceProviders\RepositoryProviders\RepositoryEnvironment;
 use Pantheon\TerminusBuildTools\ServiceProviders\CIProviders\CircleCIProvider;
-use Pantheon\TerminusBuildTools\Task\Ssh\PublicKeyReciever;
 use Symfony\Component\Console\Style\SymfonyStyle;
 
 /**
  * Project Create Command
- *
- * TODO: We could create a SiteProvider that implements PublicKeyReciever
- * rather than using $this to server that purpose.
  */
-class ProjectCreateCommand extends BuildToolsBase implements PublicKeyReciever
+class ProjectCreateCommand extends BuildToolsBase
 {
     use \Pantheon\TerminusBuildTools\Task\Ssh\Tasks;
     use \Pantheon\TerminusBuildTools\Task\CI\Tasks;
 
-    const PASSWORD_ERROR_MESSAGE="Admin password cannot contain the characters ! ; ` or $ due to a Pantheon platform limitation. Please select a new password.";
-
     /**
+     * Initialize the default value for selected options.
      * Validate requested site name before prompting for additional information.
      *
      * @hook init build:project:create
      */
-    public function validateSiteName(InputInterface $input, AnnotationData $annotationData)
+    public function initOptionValues(InputInterface $input, AnnotationData $annotationData)
```

Comment:
Admin password is not initialized from the existing value and is prompted for each time. We should grab it from the build metadata.